vault/resource_azure_secrets_sync_destination.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+)
+
+const (
+	fieldKeyVaultURI = "key_vault_uri"
+	fieldCloud       = "cloud"
+)
+
+var azureSyncDestinationFields = []string{
```

Comment:
Is there a reason why we didn't include SubscriptionID here for Azure?

Reply:
the subscription_id field wasn't ever needed to sync secrets to azure and was removed from vault as well: https://github.com/hashicorp/vault/pull/23923

Reply:
Ooo, I missed this change. Thank you!